src/registrar/management/commands/populate_organization_type.py

```diff
@@ -0,0 +1,180 @@
+import argparse
+import logging
+import os
+from typing import List
+from django.core.management import BaseCommand
+from registrar.management.commands.utility.terminal_helper import TerminalColors, TerminalHelper, ScriptDataHelper
+from registrar.models import DomainInformation, DomainRequest
+from registrar.models.utility.generic_helper import CreateOrUpdateOrganizationTypeHelper
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Loops through each valid DomainInformation and DomainRequest object and updates its organization_type value"
```

Comment:
I would clarify here what counts as a "valid" domain information and domain request object